frontend/packages/dev-console/src/components/helm/HelmReleaseUpgrade.tsx

```diff
@@ -0,0 +1,179 @@
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import { RouteComponentProps } from 'react-router';
+import { Formik } from 'formik';
+import * as yup from 'yup';
+import * as _ from 'lodash';
+import { safeDump, safeLoad } from 'js-yaml';
+import { ALL_NAMESPACES_KEY } from '@console/shared';
+import { coFetchJSON, coFetch } from '@console/internal/co-fetch';
+import { history, PageHeading, LoadingBox } from '@console/internal/components/utils';
+import NamespacedPage, { NamespacedPageVariants } from '../NamespacedPage';
+import { HELMRELEASES_ALL_NS_PAGE_URI } from './HelmReleaseDetails';
+import HelmReleaseUpgradeForm from './HelmReleaseUpgradeForm';
+import { HelmRelease, HelmChartData } from './helm-types';
+import { getChartURL } from './helm-utils';
+
+const handleNamespaceChange = (newNamespace: string): void => {
```

Comment:
The UXD doesn't have the namespace selector disabled

Reply:
But you have it disabled in your code. You are setting the variant as light there.

Reply:
If I am not wrong, the `NamespacedPageVariants` takes care of the background of the area that is below the Namespace Bar.

Reply:
Ah, my mistake. You need to pass `disabled` prop to disable it. But I am not sure what is the reasoning for UXD to enabled the change of namesapce in upgrade page when it is disabled everywhere else in other flows. Also, i am not sure how the changing the namespace of release would work in Helm. Would it delete the release in current namespace and create a new release in another? IMO it should be disabled.

Reply:
@serenamarie125 @parvathyvr Can you guys weigh in on this as to why the namespace bar is enabled in helm release upgrade form?

Reply:
@debsmita1 What is the value of having the namespace bar enabled when all it does is redirect the user to the helm release list page?

Reply:
In the Build Configs/Builds/Deployments pages, if you choose to edit a resource it will take you to the YAML editor. 
The Namespace bar is not disabled, and if you go ahead and change the namespace it will take you the resources list page. I replicated this behavior in the Upgrade Helm Release Form as well.

Reply:
> Ah, my mistake. You need to pass disabled prop to disable it. But I am not sure what is the reasoning for UXD to enable the change of namesapce in upgrade page when it is disabled everywhere else in other flows. Also, i am not sure how the changing the namespace of release would work in Helm. Would it delete the release in current namespace and create a new release in another? IMO it should be disabled.

@debsmita1 @rohitkrai03 we missed that!  It's too bad this didn't come up in the review :). The project selector bar should remain disabled with this flow.